spec/srb_json_load_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+
+require './lib/models/driver'
+
+JSON_TYPES_MAP = {
+  './lib/engines/hcktest/drivers/*.json' => AutoHCK::Models::Driver,
+}.freeze
+
+describe 'srb_json_load' do
+  Dir['./**/*.json'].each do |json_file|
+    next if json_file.include? 'jtd.json'
+
+    it json_file.to_s do
+      types = JSON_TYPES_MAP.filter_map { |json_files_pattern, type| type if File.fnmatch(json_files_pattern, json_file) }
```

Comment:
```suggestion
      type = JSON_TYPES_MAP.find { File.fnmatch(_1, json_file) }[1]
```

So that you don't need to call `first` later.

Reply:
for all JSONs without type:
```
srb_json_load ./svvp.json
      Failure/Error: type = JSON_TYPES_MAP.find { |json_files_pattern, type| File.fnmatch(json_files_pattern, json_file) }[1]
      
      NoMethodError:
        undefined method `[]' for nil:NilClass
```

Reply:
OK. Fix like:

```suggestion
      pair = JSON_TYPES_MAP.find { File.fnmatch(_1[0], json_file) }
```

The idea is to use `find` to remove `first` call.